### DISCUSSION TITLE
colorbar size narrower

### DISCUSSION THREAD

**Discussion by zndx1002070101** [mtex-toolbox/mtex#1992] [Ask Anything]
![image](https://github.com/mtex-toolbox/mtex/assets/36935034/1dfcd01c-0c2c-40f9-a2d6-6b5e8b6aeb2b)
![image](https://github.com/mtex-toolbox/mtex/assets/36935034/462326b8-6d54-43a6-9673-b02e38bc2062)

In the top image I drew two years ago, the color bar is wider. I used the code to redraw but the color bar is narrower. What is the reason? And how to reproduce the same figure as the top figure? Thanks!

**Comment by kilir** [comment]
Hi, 
a) you seem to have used a different fontsize or figure size (see mtex_settings).
b) you can control the proeprties of the colobar using the handle

```matlab
cB = mtexColorbar
% eg. change Fontsize 
cB.FontAngle = 'italic';
cB.FontSize = 20;
cB.Limits = [0 5];
cB.Position(3) = cB.Position(3) * 2;
cB.Position(4) = cB.Position(4) / 2;
cB.LineWidth = 1;
% and so on....
```

Hope that helps.
Cheers,
Rüdiger
